modular_skyrat/code/modules/mob/living/carbon/human/species.dm

```diff
@@ -12,7 +12,18 @@
 		/datum/mob_descriptor/height = "default",
 		/datum/mob_descriptor/build = "default",
 	)
-
+	var/list/pain_emote_by_power = list(
```

Comment:
```suggestion
	var/static/list/pain_emote_by_power = list(
```

Reply:
or better yet just make a switch on the proc that uses it

Reply:
Might change depending on species, so i'll keep it like that